Amplify/Categories/DataStore/Model/Internal/Schema/ModelSchema+Identifiers.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ModelSchema+Identifiers.swift
+//  
+//
+//  Created by Sapir Muallem on 23/04/2023.
+//
+
+import Foundation
+
+extension ModelSchema {
+    func getModelIdentifiers(from modelObject: [String: JSONValue]) -> [LazyReferenceIdentifier] {
```

Comment:
Just to make it read more idiomatically, how about we rename this selector to `lazyReferenceIdentifiers(from:)`?